src/app/content/highlights/reducer.ts

```diff
@@ -155,6 +155,7 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
     case getType(actions.clearFocusedHighlight): {
       return omit('focused', state);
     }
+    case getType(actions.printSummaryHighlights):
```

Comment:
i think you have to either switch on `hasMoreResults` in the component or have two actions, one for handling the loading and the printing and one just to set the loading state